content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -15,7 +15,7 @@ primary resources via the standard HTTP verbs (POST, PUT, PATCH, DELETE,
 GET).
 
 For some resources, the API includes additional subresources that allow
-fine grained authorization (such as a separating viewing details for a Pod from
+fine grained authorization (such as separating viewing details for a Pod from
```

Comment:
Slight modification because the sentence doesn't read correctly with your change. It didn't read correctly before, either.
```suggestion
fine grained authorization (such as separate viewing details for a Pod from
```